### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by Veekshith-Thoram**
image

### CODE REVIEW

File: toast/toast.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+toastBtn.addEventListener("click", displayToast);
+
+function displayToast(){
+    toastText.style.display = "block";
+    setTimeout(hide, 2000);
+}
+
+function hide(){
+    toastText.style.display="none";
+}
```
Comment (VishalPatil18) [inline toast/toast.js:14]: You could add an empty line at the end to avoid such warnings by Github

**Review by VishalPatil18 — APPROVED** [review]

File: toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+.desc-txt{
+    
+    margin-top: 0.5rem;
+    font-weight: 600;
+}
+
+.container{
+    position: relative;
+}
+.sidenav-container{
```
Comment (NamrataRaikwar2002) [inline toast/toast.css]: you should add one empty line. You can use Beauty File for formatting.

File: toast/toast.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+}
+.links:hover{
+    color: var(--primary-color);
+}
+.side-nav-link{
+    margin-top: 1rem;
+}
+
+
+
```
Comment (NamrataRaikwar2002) [inline toast/toast.css]: You can remove these empty lines.

**Review by NamrataRaikwar2002 — APPROVED** [review]
Good  work Veekshith, gave some suggestion.

**Pull request merged**